src/cfnlint/core.py

```diff
@@ -93,9 +93,12 @@ def create_parser():
 
     # Alllow the template to be passes as an optional or a positional argument
     parser.add_argument(
-        'template', nargs='?', help='The CloudFormation template to be linted')
+        'templates', nargs='*', default=[],
+        help='The CloudFormation templates to be linted')
     parser.add_argument(
-        '-t', '--template', dest='template_alt', help='The CloudFormation template to be linted')
```

Comment:
I have to check all the editor plugins.  I know for atom, vs code this will break them.  I would like to keep `--template` available until we can get them converted to `-t-` or `--templates` and give a little bit of time for people to update.